config/api/src/main/java/org/jboss/windup/config/tags/TagsSaxHandler.java

```diff
@@ -0,0 +1,84 @@
+package org.jboss.windup.config.tags;
+
+import java.util.Stack;
+import java.util.logging.Logger;
+import org.apache.commons.lang3.StringUtils;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+
+/**
+ * Expects the following XML structure:
+ *
+ * {tag name="foo"}
+ *    {tag name="bar" root="true"}
+ *    {tag name="baz"}
+ *    ...
+ * {tag name="bar"}
+ *    {tag name="boo"  parents="bar baz"}
+ *    ...
+ */
+public class TagsSaxHandler extends DefaultHandler
+{
+    private static final Logger log = Logger.getLogger(TagsSaxHandler.class.getName() );
+
+    private final TagService tagService;
+    private final Stack<Tag> stack = new Stack<>();
+
+
+    public TagsSaxHandler(TagService tagsService)
+    {
+        this.tagService = tagsService;
+    }
+
+
+
+    @Override
+    public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException
```

Comment:
Joox?  I think this is quite simple structure which you would parse using a stack anyway...